vipps-recurring-api.md

```diff
@@ -699,8 +700,8 @@ the charge states returned by
 
 | # | State      | Description                                                                          |
 |:--|:-----------|:-------------------------------------------------------------------------------------|
-| 1 | `PENDING`  | The charge has been created, but has not yet been approved by the user. |
-| 2 | `DUE`      | The charge is now visible in Vipps, and will be made in the next 30 days (this was previously 6). |
```

Comment:
Saying anything about when the charge will be processed here is inaccurate. Charges are processed on the dueDate and until dueDate+retryDays. I think the intention was to explain that charges get this status 30 days before dueDate. I can add that if it seeems necessary

Reply:
Maybe write that explicitly, as some merchants seem confused.